index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Jonathan Arias Gallego | Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <nav id="navbar" class="open">
+      <div class="navbar__status">
+        <img src="Assets/Time Style.svg" alt="">
+        <img src="Assets/Bars.svg" alt="">
+      </div>
+      <div class="toolbar">
+        <a href="#" class="toolbar__logo">My Logo</a>
+        <a href="#" id="toolbar__button"><img src="Assets/Icon - Menu.svg" alt=""></a>
+        <div class="toolbar__menu">
+          <a href="#porfolio" class="">Portfolio</a>
+          <a href="#about" class="">About</a>
+          <a href="#contact" class="">Contact</a>
+          <a href="#" class="menu__button"><img src="Assets/msg-box.svg" alt=""></a>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section id="portfolio">
+      <h1>Hey There. I’m Alex.</h1>
+      <h2>I'm a Software Developer</h2>
+      <p class="portfolio__paragraph">I can help you build a product , feature or website Look through some of my
+        work and experience! If you
+        like what you see and have a project you need coded,
+        don’t hesitate to contact me.
+      </p>
+      <div class="social-media">
+        <a href=""><img src="Assets/Github.svg" alt=""></a>
+        <a href=""><img src="Assets/Linkedin.svg" alt=""></a>
+        <a href=""><img src="Assets/Icon - Angelist.svg" alt=""></a>
+        <a href=""><img src="Assets/Twitter.svg" alt=""></a>
+        <a href=""><img src="Assets/Medium.svg" alt=""></a>
+      </div>
```

Comment:
- [ ] Hey Jonathan, you're doing an awesome work so far but for semantic reasons I highly advise you place the list of icons using a `ul` tag to group them, and `li` per icon, here's how it should look like when done 👇
```html
<ul>
  <li><a href=""><img src="Assets/Github.svg" alt=""></a></li>
  <li><a href=""><img src="Assets/Linkedin.svg" alt=""></a></li>
  <li><a href=""><img src="Assets/Icon - Angelist.svg" alt=""></a></li>
  <li><a href=""><img src="Assets/Twitter.svg" alt=""></a></li>
  <li><a href=""><img src="Assets/Medium.svg" alt=""></a></li>
</ul>
```